src/sniffles/sniffles.py

```diff
@@ -183,14 +183,19 @@ def start_generation(sconf):
     # for recording how packets are generated
     fd_result = open(sconf.getResultFile(), 'w')
 
+    if allrules:
+        i = random.randrange(len(allrules))
```

Comment:
i is not a very good variable name in this instance.  rule_cursor, or rule_index, or something like that would be better.